src/components/Settings/Settings.tsx

```diff
@@ -103,6 +105,17 @@ export class Settings extends Component<Props, State> {
         }
     };
 
+    getDeviceByLabel = (label: string, deviceType: ResourceTypes) => {
+        if (deviceType === ResourceTypes.AUDIO) {
+            return this.audioDevicesInfo.find((device: MediaDeviceInfo) => {
+                return device.label === label;
+            });
+        }
```

Comment:
I prefer to see code that it's more explainable.
Adding else with the ResourceTypes.AUDIO will be better.
Also if for some reason in the future we will have new ResourceType the code will not cause any bugs.

You have it in other line.